tests/preprocessing/test_positionfixes.py

```diff
@@ -45,8 +45,14 @@ def example_positionfixes():
 
 
 @pytest.fixture
-def isolated_positionfixes():
-    """Positionfixes with two isolated positionfixes that have the same geometry but different timestamps."""
+def example_positionfixes_isolated():
+    """
+    Positionfixes with isolated positionfixes.
+
+    User1 have the same geometry but different timestamps.
+    User2 have different geometry and timestamps.
+    User3 have only one isolated positionfixes.
+    """
```

Comment:
p3 and p4 below are the same. If this is intentional I would rather only use p3 below

Reply:
strange legacy issue. Already changed